Dialogs/ProfessionDialog.hpp

```diff
@@ -2,26 +2,17 @@ class IntProfessionDialog
 {
     idd = 2999;
     movingenable = false;
-    
-    class Objects 
-    {
-		class RscModel_2230 : RscObject
```

Comment:
I do not really like this kind of name standards. Could it be a bit more expressive to reveal a bit more information? How is this class/object intended to be used? Or, what is it? What about VehicleImageContainer?